### PR TITLE
Test against newer Node.js versions

### PR DESCRIPTION
**Pull request by MarkHerhold**
Updates travis.yml, nails down test dependencies:

- `xo` is now set to the version that was released the last time the project was updated
- `ava` is now set to the current latest version (as of time of writing).

### CODE REVIEW

**Comment by SamVerschueren** [comment]
If we decide to drop tests for older versions, we should ES2015ify the code and use const, let, arrow functions etc. But that's up to @sindresorhus to decide :).

**Comment by sindresorhus** [comment]
Thanks, but easier to do this myself as there are other changes I want to do too.

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 language: node_js
 node_js:
-  - 'stable'
-  - '0.12'
-  - '0.10'
+  - 8
+  - 6
+  - 4
```
Comment (sindresorhus) [inline .travis.yml:5]: Semver specifiers are not numbers. These should be quoted.

**Comment by MarkHerhold** [comment]
Ok, sure @sindresorhus. I'd love to see this brought back up to date! Thank you!

**Pull request closed without merging**